### ISSUE TITLE
✅ ♿ 🐞 Carousel Swiper fails accessibility on Pagespeed by having no aria-label even if the property is present

### ISSUE THREAD

**Issue by Dionnie123** [elementor/elementor#28355]
### Prerequisites

- [X] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [ ] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

button, link, and menuitem elements do not have accessible names.

![image](https://github.com/user-attachments/assets/1bbc1833-47e0-40e1-84f4-a7f9b95aaeb3)

But the arial-label is present upon inspection

![image](https://github.com/user-attachments/assets/94a1495e-538a-4474-9880-73cb6e5105ba)

### Steps to reproduce

1. Create a Page with Carousel
2. Analyze the Page on https://pagespeed.web.dev/
3. Check the Accessbility Result.

### Expected behavior

The Accessibility problem should not be present because arial-label exists!

### Isolating the problem

- [ ] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [ ] This bug happens with the Hello Elementor theme active.
- [X] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

<details>
<summary>Click to reveal</summary>

````txt

== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: MySQL Community Server (GPL) v5.7.41
	PHP Version: 7.4.33
	PHP Memory Limit: 800M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 1280M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.6.1
	Site URL: https://a1loanhub.com.au
	Home URL: https://a1loanhub.com.au
	WP Multisite: No
	Max Upload Size: 1 GB
	Memory limit: 40M
	Max Memory limit: 800M
	Permalink Structure: /%postname%/
	Language: en_US
	Timezone: 0
	Debug Mode: Active

== Theme ==
	Name: Hello Elementor Child
	Version: 2.0.0
	Author: Elementor Team
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 3.1.1
	Parent Theme Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: en-US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.0 Safari/537.36

== Active Plugins ==
	Elementor
		Version: 3.23.4
		Author: Elementor.com

	Elementor Pro
		Version: 3.23.3
		Author: Elementor.com

	Enable Media Replace
		Version: 4.1.5
		Author: ShortPixel

	Real Media Library (Free)
		Version: 4.22.22
		Author: devowl.io

	WPS Hide Login
		Version: 1.9.16.7
		Author: WPServeur, NicolasKulka, wpformation


== Elements Usage ==
	
	archive : 1
		archive-posts : 1
		container : 2
		heading : 1
	footer : 2
		container : 28
		form : 2
		heading : 13
		icon-list : 1
		nav-menu : 10
		text-editor : 8
		theme-site-logo : 2
	header : 1
		button : 1
		container : 3
		heading : 1
		icon : 1
		nav-menu : 1
		theme-site-logo : 1
	popup : 1
		container : 1
		nav-menu : 1
		theme-site-logo : 1
	wp-page : 18
		button : 19
		container : 280
		divider : 2
		form : 1
		google_maps : 1
		heading : 161
		html : 8
		icon-box : 6
		icon-list : 15
		image : 69
		nested-accordion : 2
		nested-carousel : 1
		nested-tabs : 1
		posts : 1
		spacer : 7
		text-editor : 105
	single-post : 1
		container : 3
		heading : 1
		post-info : 1
		post-navigation : 1
		theme-post-content : 1
		theme-post-featured-image : 1
		theme-post-title : 1


== Settings ==
	
	cpt_support: post, page
	allow_tracking: yes
	font_display: swap


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	
	recaptcha_v3: Active


== Elementor Experiments ==
	Improved CSS Loading: Active by default
	Inline Font Icons: Inactive by default
	Additional Custom Breakpoints: Active by default
	Flexbox Container: Active by default
	Grid Container: Active by default
	Upgrade Swiper Library: Active by default
	Nested Elements Performance: Active by default
	Optimized Control Loading: Inactive by default
	Optimized Markup: Inactive by default
	Plugin Onboarding: Active by default
	Default to New Theme Builder: Active by default
	Header &amp; Footer: Inactive
	Elementor Home Screen: Active by default
	Editor Top Bar: Inactive by default
	Build with AI: Active by default
	Landing Pages: Inactive
	Nested Elements: Active
	Lazy Load Background Images: Active
	Element Caching: Active
	Link In Bio: Inactive by default
	Floating Buttons: Inactive by default
	Elementor Editor Events: Inactive by default
	Display Conditions: Active by default
	Form Submissions: Active by default
	Menu: Inactive by default
	Taxonomy Filter: Active by default
	Search: Inactive by default


== Log ==
	
Log: showing 20 of 282024-01-08 04:47:32 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.17.1',
  'to' => '3.18.2',
)]
2024-01-08 04:47:32 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.17.1',
  'to' => '3.18.2',
)]
2024-01-08 07:57:16 [info] Uploading Kit:  [array (
  'kit_id' => 'undefined',
  'referrer' => 'local',
)]
2024-01-08 07:57:25 [info] Selected import runners: site-settings, plugins, templates, taxonomies, elementor-content, wp-content, elements-default-values 
2024-01-08 07:57:26 [info] Import runner completed: site-settings ✓ 
2024-01-08 07:57:27 [info] Import runner completed: plugins ✓ 
2024-01-08 07:57:29 [info] Import runner completed: templates ✓ 
2024-01-08 07:57:29 [info] Import runner completed: taxonomies ✓ 
2024-01-08 07:57:30 [info] Import runner completed: elementor-content ✓ 
2024-01-08 07:57:31 [info] Import runner completed: wp-content ✓ 
2024-08-06 07:45:53 [info] elementor::elementor_updater Started 
2024-08-06 07:45:53 [info] Elementor/Upgrades - _on_each_version Start  
2024-08-06 07:45:53 [info] Elementor/Upgrades - _on_each_version Finished 
2024-08-06 07:45:53 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.18.3',
  'to' => '3.23.4',
)]
2024-08-06 07:45:53 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.18.3',
  'to' => '3.23.4',
)]
2024-08-12 04:11:45 [info] elementor-pro::elementor_pro_updater Started 
2024-08-12 04:11:45 [info] Elementor Pro/Upgrades - _on_each_version Start  
2024-08-12 04:11:45 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2024-08-12 04:11:45 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.18.2',
  'to' => '3.23.3',
)]
2024-08-12 04:11:45 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.18.2',
  'to' => '3.23.3',
)]

PHP: showing 3 of 3PHP: 2024-01-08 04:46:07 [notice X 3][../wp-content/plugins/elementor-pro/modules/theme-builder/widgets/site-logo.php::192] Undefined index: options [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-03-16 10:08:54 [notice X 1][../wp-content/plugins/elementor-pro/modules/forms/classes/recaptcha-v3-handler.php::124] Undefined index: score [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-08-12 08:20:51 [notice X 2][../wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::182] Undefined index: editor_post_id [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible

== Elementor Pro - Compatibility Tag ==

````

</details>

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by Dudeney95** [comment]
+1

**Comment by nicholaszein** [comment]
Hello @Dionnie123! 

Thank you for submitting your issue! 🙏 

🏷️ We've labeled this issue and our team will take a look at it. If there is indeed a problem, they'll work on fixing it. We can't say exactly when it'll be sorted out, but we kindly ask for your patience. Thanks for understanding!

Best regards

**Comment by danvy** [comment]
I have the same problem.
This prevent us to be accessibility compliant 😒
<img width="955" alt="image" src="https://github.com/user-attachments/assets/72eae692-420e-4a7a-8391-dcb94f44fbd7">
You can check the Google PageSpeed Insights report here https://pagespeed.web.dev/analysis/https-altoviz-com-fr/9vrgz6a7sg?form_factor=desktop

**Comment by ricardovigatti** [comment]
+1

**Comment by fabbea** [comment]
Hi,
Exactly the same problem with Elementor pro and Hello Theme.
Regards

**Comment by rami-elementor** [comment]
From the screenshots I see that @Dionnie123 is using the nested "Carousel" widgets and @danvy is using the "Image Carousel" widget.

We plan to fix several carousels-related accessibility issues in Elementor 3.26, this issue will be resolved too.

**Comment by rampatel5778** [comment]
Hello,

I found an issue with the navigation and pagination in both the slider and loop grid.

    Navigation: The role="button" is added but without an aria-label, causing accessibility issues.
    Pagination (bullets): The aria-label is included, but the role is missing.

Both issues are affecting the page speed and accessibility score.

I have attached screenshots for reference.
![Screenshot 2024-11-28 at 16-02-59 PageSpeed Insights](https://github.com/user-attachments/assets/3afa3732-8fc2-4041-87ec-8668de9216c8)

Thank you for looking into this!

**Comment by rami-elementor** [comment]
@rampatel5778 Which Elementor version did you use?

**Comment by nicholaszein** [comment]
Hello there everyone!

We have great news! 🙌 

### 📢 We're happy to announce that this issue was resolved with the release of Elementor Pro v3.26.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂

----
PS: If you still experience the same issue after updating to the latest version, and can reproduce it consistently even after troubleshooting it and clearing all levels of cache, please [open a new Bug Report](https://elemn.to/gh-new-bug-report) with detailed information so we can investigate it further.
